src/encoded/static/components/genome_browser.js

```diff
@@ -549,14 +560,15 @@ class GenomeBrowser extends React.Component {
             let labelLength = 0;
             const defaultHeight = 34;
             const extraLineHeight = 12;
-            const maxCharPerLine = 30;
```

Comment:
There are still some cases with overflow which should be fixed:

![Screen Shot 2020-10-06 at 1 37 53 PM](https://user-images.githubusercontent.com/2887343/95260014-e3bdd300-07dc-11eb-8096-9bea106f297e.png)

Reply:
How should this be fixed? Make `maxCharPerLIne` even smaller?

Reply:
How did you get the track labels to not show the expander icons?

Reply:
OK, I see now where that depends on a lot of things. I updated the height.